modules/dumbCommandModule/index.js

```diff
@@ -73,6 +73,15 @@ module.exports.commands = {
       if (typeof commandDict[parts[0]] === 'undefined') return reply('No such command');
       reply(`Blame ${commandDict[parts[0]].blame} for this`);
     },
+    rblame(r, parts, reply) {
+      if (parts.length === 0) return reply('please specify a user to blame');
```

Comment:
`if (parts.length !== 1)` to match the other branches, if you're not going to use multiple parts